pkg/apis/go.mod

```diff
@@ -3,8 +3,12 @@ module github.com/rancher/rancher/pkg/apis
 go 1.20
 
 replace (
-	// wrangler bracnhes need to be updated before replace can be removed
-	github.com/rancher/wrangler v1.1.1 => github.com/rancher/wrangler v1.1.1-0.20230831050635-df1bd5aae9df
+	github.com/rancher/aks-operator v1.2.0 => github.com/KevinJoiner/rancher-aks-operator v0.0.0-20240124143758-627d80adf4e6
+	github.com/rancher/eks-operator v1.3.0 => github.com/KevinJoiner/rancher-eks-operator v0.0.0-20240123183955-8f2f749b3ed1
```

Comment:
Remove and update commit when https://github.com/rancher/eks-operator/pull/335 is merged.